sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ReceiverManager.cs

```diff
@@ -12,59 +12,44 @@
 namespace Azure.Messaging.ServiceBus
 {
     /// <summary>
-    /// Represents a single receiver instance that multiple threads spawned by the
+    /// Represents a single receiver instance that multiple tasks spawned by the
     /// <see cref="ServiceBusProcessor"/> may be using to receive and process messages.
     /// The manager will delegate to the user provided callbacks and handle automatic
     /// locking of messages.
     /// </summary>
     internal class ReceiverManager
     {
         protected virtual ServiceBusReceiver Receiver { get; set; }
-        protected readonly ServiceBusConnection _connection;
-        protected readonly string _fullyQualifiedNamespace;
-        protected readonly string _entityPath;
-        protected readonly string _identifier;
+
+        protected readonly ServiceBusProcessor Processor;
```

Comment:
As part of the public API surface, it would be helpful if the `protected` member had doc comments to help developers extending the type to understand what role they play.

Reply:
ReceiverManager is internal.